plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Shipping.php

```diff
@@ -165,6 +194,6 @@ private static function is_selling_digital_type_only() {
 		$profiler_data = get_option( OnboardingProfile::DATA_OPTION, array() );
 		$product_types = isset( $profiler_data['product_types'] ) ? $profiler_data['product_types'] : array();
 
-		return [ 'downloads' ] === $product_types;
+		return array( 'downloads' ) === $product_types;
```

Comment:
@moon0326 I'm not really familiar with PHP. Is there any difference between `[]` and `array`, except for [PHP < 5.4 backward-compatibility](https://stackoverflow.com/questions/26651996/php-array-vs-in-method-and-variable-declaration).

Reply:
@chihsuan That's actually the same. It's just a different syntax and WP recommends using array. I think PHP CS fixed it automatically for me.